lnwallet/chainfee/estimator.go

```diff
@@ -476,17 +476,24 @@ func (b *BitcoindEstimator) fetchEstimate(confTarget uint32) (SatPerKWeight, err
 	// estimated fee rate from its sat/kb representation to sat/kw.
 	satPerKw := SatPerKVByte(satPerKB).FeePerKWeight()
 
+	// Bitcoind will not report any fee estimation if it has not enough
+	// data available hence the fee will remain zero. We return an error
+	// here to make sure that we do not use the min relay fee instead.
+	if satPerKw == 0 {
```

Comment:
Sounds reasonable, I wonder whether we need to Refactor `bitcoindConn *rpcclient.Client` to interface it out so that we can mock the rpc call, or do you maybe have other idea ?

Reply:
It'd be great to have an interface there, yes, but maybe a future TODO to limit the scope.